content/blog/2018-12-11-rcites.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: "accessing-CITES-data"
+title: rcites - The story behind the package
+package_version: 1.0.0
+authors:
+  - name: Ignasi Bartomeus
+    url: https://bartomeuslab.com/
+    twitter: ibartomeus
+  - name: Kevin Cazelles 
+    url: https://kevcaz.github.io/
+    twitter: KCazelles
+  - name: Jonas Geschke
+    url:  https://www.linkedin.com/in/jonasgeschke/
+    twitter: J_Geschke
+date: 2018-12-11
+categories: blog
+topicid: XXX 
+Tags:
+- r
+- package
+- data-access
+- API-client
+- cites 
+---
+
+### The Ecology Hackathon
+
+Almost one year ago now, ecologists filled a room to meet for an “Ecology Hackathon: Developing R Packages for Accessing, Synthesizing and Analyzing Ecological Data” that was part of the BES, GfÖ, NecoV, and EEF Joint Annual Meeting 2017 “Ecology Across Borders” in Ghent, co-organised by rOpenSci and Methods in Ecology and Evolution (see [Methodsblog entry 1](https://methodsblog.com/2017/11/16/hackathon-challenges/) and [Methodsblog entry 2](https://methodsblog.com/2018/01/23/ecology-hackathon/). At different tables, different people joined each other to work on different ideas to implement as R packages. At our table, we were around ten people that more or less did not know anything about what we aimed for. We barely knew each other and nobody had clear expectations, just the desire of learning more about R packages. As a piece of paper on our table indicated, we were interested in a common idea posted as a [wishlist](https://github.com/ropensci/wishlist/issues/29) in the rOpenSci community: building an R package to interact with [CITES](https://cites.org/) and its [Speciesplus database](https://speciesplus.net). At 10 am, nobody had a clear idea on where to start. By 6 pm, we had a functional prototype of the rcites package, which was really rewarding and gave motivation to follow up on the package development. We did a great team-work, met new researchers, learn a bunch of new stuff, this was definitely a successful hackathon!   
```

Comment:
"We did a great team-work, met new researchers, learn a bunch of new stuff, this was definitely a successful hackathon! "

some grammar changes:

We did great team-work, met new researchers, and learned a bunch of new stuff. This was definitely a successful hackathon!